src/index.tsx

```diff
@@ -40,6 +41,14 @@ const initialState = {};
 const store = configureStore(initialState);
 const queryClient = new QueryClient();
 
+if (process.env.REACT_APP_GTM_ID) {
+  const tagManagerArgs = {
+    gtmId: process.env.REACT_APP_GTM_ID || '',
```

Comment:
this is redundant since you are checking earlier if it's defined: `process.env.REACT_APP_GTM_ID || ''`